cmds/core/wget/wget.go

```diff
@@ -91,8 +87,16 @@ func run() (reterr error) {
 	return nil
 }
 
+func defaultOutputPath(url *url.URL) string {
+	if url.Path != "" && url.Path[len(url.Path)-1] != '/' {
```

Comment:
would strings.Suffix work here? 
I think it would and it makes this simpler
https://go.dev/play/p/7TyYCf1DmRe

Reply:
Updated.